package.json

```
@@ -17,6 +17,7 @@
     "e2e:fast": "playwright test --retries=2 --workers=3 -x --trace on",
     "e2e:debug": "playwright test --debug -x --trace on /test",
     "e2e:watch": "playwright test --ui",
+    "e2e:codegen": "playwright codegen localhost:3000/drilldown",
```

```suggestion
    "e2e:codegen": "playwright codegen",
```

We can always pass parameters to the npm script so that it works for everyone in the team. For instance, to test WingMan I'm using:

`npm run e2ecodegen -- http://localhost:3001/a/grafana-metricsdrilldown-app`